homeassistant/components/mqtt/util.py

```diff
@@ -111,3 +113,10 @@ def valid_publish_topic(value: Any) -> str:
     },
     required=True,
 )
+
+
+def get_mqtt_data(hass: HomeAssistant, ensure_exists: bool = False) -> MqttData:
```

Comment:
I originally placed this in `models.py`, but I think it fits better here... as it's an internal utility to get the model instance.